roles/wp-cli/defaults/main.yml

```diff
@@ -1,4 +1,5 @@
 wp_cli_version: 1.5.1
+wp_cli_phar_checksum: 'sha512:8dd68c98c6fa00e1acc5e036f9393c8b052937045b5232e4aa0eb4f15773908eae48760607bc853a4f951bd5ba69e5050337e5d9dcfa48df87a12cebb1de3432'
```

Comment:
Can this be double quotes for consistency? 👼

Reply:
done